grant/projects/kton_staking_ui.md

```diff
@@ -0,0 +1,48 @@
+## Project name
+
+Darwinia KTON Staking UI
+
+### Project Description
+
+This project is a frontend user interface for the new KTON Staking. [DIP-5](https://github.com/darwinia-network/DIPs/pull/15) proposes some adjustments to the KTON staking, which will separate the KTON Staking from Collator Staking while keeping incentive for ring pool and kton pool unchanged. [ref](https://github.com/orgs/darwinia-network/discussions/1393)
+
+### **Why is this project good for the Darwinia ecosystem**
+
+This project will provide a clear and easy KTON Staking web UI to users.
+
+Without this UI, users will manually interact with the new KTON Staking contract. This could be a big obstacle for non-tech users to stake their KTON. More convenient staking can not only help KTON holders obtain more rewards, but will also make Darwinia Chain more active.
+
+### Team
+
+- Snoopy1412([@snoopy1412](https://github.com/snoopy1412))
+
+### Team Code Repos
+
+- [https://github.com/snoopy1412/kton-stake](https://github.com/snoopy1412/kton-stake)
+
+### Development Roadmap
+
+### **Phase 1**
+
+Estimated Duration: 2 weeks
+
+Cost: 1000 USD
+
+#### Tasks:
+
+1. **Interface Design and Responsive Implementation**
+
+   - Design an intuitive and user-friendly interface, adopting the latest UI/UX design concepts.
+   - Use React and TailwindCSS for responsive design, ensuring the application provides a consistent user experience across various devices.
+
+2. **Web3 Integration**
+
+   - Integrate Web3 technology using the Wagmi library and `@rainbow-me/rainbowkit` for blockchain interactions, enabling users to manage assets and execute transactions.
```

Comment:
With this integration, wallet connect is also supported, right?

Reply:
> With this integration, wallet connect is also supported, right?

Yes, Wallet Connect is supported because I have integrated RainbowKit